DelvUI/Interface/Jobs/WhiteMageHud.cs

```diff
@@ -215,14 +216,14 @@ public new static WhiteMageConfig DefaultConfig()
         }
 
         [NestedConfig("Lily Bar", 30)]
-        public ChunkedBarConfig LilyBar = new ChunkedBarConfig(
+        public LilyBarConfig LilyBar = new LilyBarConfig(
             new(-64, -32),
             new(126, 20),
             new PluginConfigColor(new(0f / 255f, 64f / 255f, 255f / 255f, 100f / 100f))
         );
 
         [NestedConfig("Blood Lily Bar", 35)]
-        public ChunkedBarConfig BloodLilyBar = new ChunkedBarConfig(
+        public BloodLilyBarConfig BloodLilyBar = new BloodLilyBarConfig(
```

Comment:
Or these?

Reply:
I was expecting I'd need to add converter stuff, but everything worked great as is even going back and forth between the builds